### ISSUE TITLE
Possible Error

### ISSUE THREAD

**Issue by some-vec-byte** [abfo/shapefile#2]
In Shapefile.cs Line 172:
_boundingBox = new RectangleD(_mainHeader.XMin, _mainHeader.YMin, _mainHeader.XMax, _mainHeader.YMax);

Constructor of RectangleD:
(double left, double top, double right, double bottom)

so shouldnt it be:
_boundingBox = new RectangleD(_mainHeader.XMin, _mainHeader.YMax, _mainHeader.XMax, _mainHeader.YMin);

Greetings

**Comment by abfo** [comment]
Yes, I've noticed this and it doesn't look right. I have a lot of code that depends on the current implementation and there are many other users of this library that might not welcome this kind of breaking change. Maybe I should add a flag to allow this to be changed. It's on my list to look at.
